src/System.ComponentModel.Primitives/pkg/System.ComponentModel.Primitives.pkgproj

```diff
@@ -2,8 +2,11 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <ProjectReference Include="..\ref\4.1.0\System.ComponentModel.Primitives.depproj">
```

Comment:
Can you please have a read over my document that talks about how to version these libraries and make the necessary updates and/or leave comments on my PR (https://github.com/dotnet/corefx/pull/12036) about followup questions.